### PR TITLE
Feedback Formular

### PR DESCRIPTION
**Pull request by robert-kub**

### CODE REVIEW

File: functions_actions.php [hunk 1]
```diff
@@ -301,3 +301,45 @@ function actionReport($label)
...
+        );
+            
+        if (!$gesendet) {
+            http_response_code(500);
+        }
+    }
+
+    if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+        http_response_code(400);
+    }
```
Comment (xi) [inline functions_actions.php:342]: Why return a 400 status on POST?

File: functions_mail.php [hunk 2]
```diff
@@ -122,6 +122,24 @@ function send_reminder($to, $name, $id, $hash, $label)
...
+    $body = $label['Feedback_frage1'].": ".$_POST['frage1']."\n".
+            $label['Feedback_frage2'].": ".$_POST['frage2']."\n".
+            $label['Feedback_frage3'].": ".$_POST['frage3']."\n".
+            $label['Feedback_frage4'].": ".$_POST['frage4']."\n".
+            $label['Feedback_frage5'].": ".$_POST['frage5']."\n".
+            $label['Feedback_frage6'].": ".$_POST['frage6']."\n".
+            $label['Feedback_frage7'].": ".$_POST['frage7']."\n";
+    print $to;
+    print $subject;
+    print $body;
```
Comment (xi) [inline functions_mail.php:141]: Should these actually be here or are they just leftovers from development?

File: templates/forms/feedback.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<form method="POST">
+    <input type="hidden" name="csrf_token" value="<?php e($GLOBALS['csrf_token']) ?>" />
+
+    <p><?php e($label['Feedback_intro']) ?></p>
+
+        <p>
+            <label for="feedback_frage1"><?php e($label['Feedback_frage1']) ?></label></br>
```
Comment (xi) [inline templates/forms/feedback.php:7]: No element with id `feedback_fragen1` exists. Could we use

```html
<label>
  <input type="checkbox" name="frage1" <?php e($_POST['frage1']) == 'ja' ? 'checked' : '' ?>>
  <?php e($label['Feedback_frage1']) ?>
</label>
```

instead?

File: functions_actions.php (hunk 1 above)
Comment (robert-kub) [reply functions_actions.php:342]: Can be removed. Just copied it from the action above.

File: functions_mail.php (hunk 2 above)
Comment (robert-kub) [reply functions_mail.php:141]: prints can be removed

File: templates/forms/feedback.php (hunk 3 above)
Comment (robert-kub) [reply templates/forms/feedback.php:7]: yes, that is better.

Comment (robert-kub) [reply templates/forms/feedback.php:7]: There should be a yes/no choice. I will keep the radio-buttons.

**Pull request closed without merging**